.gitignore

```diff
@@ -34,3 +34,4 @@ venv*/
 *.wp[ur]
 
 tests/apps/verify-*/
+logs/
```

Comment:
What would create a logs directory that isn't already ignored by one of the other rules? I tried running the unit tests, but they didn't do it.

Reply:
I did a manual build of the verify-toga app, to replicate one of the failures I saw during the release process. That generated a logs folder in tests/apps.

Reply:
But the tests/apps subdirectories are already ignored by the line above.

Reply:
Yes, but the logs folder was generated in `test/apps`, because it was `briefcase new` that failed, rather than `create` etc. I guess we could narrow it to that specific known failure; but I figured we might as well have a blanked rule to avoid any stray logs folders.